shared/aux.go

```diff
@@ -352,3 +357,27 @@ func fileExists(files []os.DirEntry, workload string) bool {
 
 	return false
 }
+
+// VerifyFileMatchWithPath verify actual file list matches the expected file list
+func VerifyFileMatchWithPath(actualFileList []string, expectedFileList []string, path string) {
+	equalLength := Expect(len(actualFileList)).To(Equal(len(expectedFileList)))
```

Comment:
on this pkg we do not use gomega or ginkgo as per architectural standars

Reply:
fixed